src/state.rs

```diff
@@ -1708,5 +1752,12 @@ mod tests {
             vec!["cluster1//v1/Pod/default/wl_match"],
             "failover full match selects closest match",
         );
+        assert_not_endpoint(
+            &wl_no_locality,
```

Comment:
```suggestion
            &wl_empty_ip,
```

Reply:
This is testing sending from `wl_no_locality` as the src

Reply:
oh, right